app/resources/css-default.template

```diff
@@ -194,6 +196,23 @@ input {
   font-size: 13px;
   margin-bottom: .5em;
   max-width: 100%;
+  border: 0;
+  outline: 0;
+  border-bottom: 1px solid rgb(169, 169, 169);
+  background: transparent;
+}
+
+input:focus {
+  border: 0;
+  outline: 0;
+  border-bottom: 2px solid #3b78e7;
+}
+
+input:not([type]):disabled {
```

Comment:
this is for /* Show disabled input states with dot line */
![disable](https://cloud.githubusercontent.com/assets/15961861/18829258/6e91e2b8-8416-11e6-858c-37b92ac85525.png)